tests/SprykerTest/Client/RabbitMq/_support/Helper/RabbitMqHelper.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerTest\Client\RabbitMq\Helper;
+
+use Codeception\Module;
+use Codeception\Util\Stub;
+use Generated\Shared\Transfer\QueueConnectionTransfer;
+use Generated\Shared\Transfer\StoreTransfer;
+use PhpAmqpLib\Channel\AMQPChannel;
+use Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientBridge;
+use Spryker\Client\RabbitMq\Model\Connection\Connection;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigFilter\ConnectionConfigFilter;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigMapper\ConnectionConfigMapper;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionCreator\ConnectionCreator;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionManager;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionManagerInterface;
+use Spryker\Client\RabbitMq\RabbitMqConfig;
+
+class RabbitMqHelper extends Module
+{
+    protected const STORE_NAME = 'DE';
+    protected const LOCALE_CODE = 'en_US';
```

Comment:
not a blocker, but I would prefer not to have a hardcoded values for new tests.

Reply:
In this case, it doesn't matter. Here also can be `locale` instead of `en_US`. I just gave these constants meaningful values.